pkg/agent/openflow/pipeline.go

```diff
@@ -192,6 +195,30 @@ func (c *client) gatewayClassifierFlow(gatewayOFPort uint32, category cookie.Cat
 		Done()
 }
 
+// patchPortClassifierFlow generates the flow to mark IP traffic comes from the patch port and
+// are destined to local pods
+func (c *client) patchPortClassifierIPFlow(patchPort uint32, podIP net.IP) binding.Flow {
+	classifierTable := c.pipeline[classifierTable]
+	return classifierTable.BuildFlow(priorityNormal).
+		MatchInPort(patchPort).
+		MatchDstIP(podIP).
+		Action().LoadRegRange(int(marksReg), markTrafficFromPatchPort, binding.Range{0, 15}).
+		Action().ResubmitToTable(classifierTable.GetNext()).
+		Done()
+}
+
+// patchPortClassifierFlow generates the flow to mark ARP traffic comes from the patch port and
+// are destined to local pods
+func (c *client) patchPortClassifierARPFlow(patchPort uint32, podIP net.IP) binding.Flow {
+	classifierTable := c.pipeline[classifierTable]
+	return classifierTable.BuildFlow(priorityNormal).
+		MatchInPort(patchPort).
```

Comment:
It would be better to explicately add `MatchProtocol(binding.ProtocolARP)`

Reply:
that is good point, will do.

Reply:
added binding arp and ip protocol, done.